view/adminhtml/templates/boltpay/button.phtml

```diff
@@ -20,5 +20,32 @@ $customerCreditCardInfos = $block->getCustomerCreditCardInfo();
     <input type="hidden" id="bolt-billing-address" value='<?php echo $block->getBillingAddress() ?>' />
     <input type="hidden" id="bolt-place-order-payload" value='<?php echo $block->getPlaceOrderPayload() ?>' />
     <div class="bolt-checkout-button with-cards"></div>
-    <input type="hidden" class="required-entry">
+    <input type="hidden" class="required-entry" id="bolt-required-field">
 </fieldset>
+
+<?php if($customerCreditCardInfos): ?>
+    <script>
+        require(['jquery', 'jquery/ui'], function($){
+            $(document).on('change', 'select[name="credit-cards"]', function(){
```

Comment:
Perhaps this select should have more unique name, for instance with "bolt-" prefix.